library/core/src/main/java/com/google/android/exoplayer2/upstream/HttpDataSource.java

```diff
@@ -294,15 +295,22 @@ public InvalidContentTypeException(String contentType, DataSpec dataSpec) {
      */
     public final int responseCode;
 
+    /**
+     * The HTTP status message.
+     */
+    public final @Nullable String responseMessage;
```

Comment:
```suggestion
    @Nullable public final String responseMessage;
```

Reply:
I was actually looking at `OkHttpDataSource` to put the `@Nullable` at the right position :)